languages/go/exercises/concept/numbers/.docs/introduction.md

```diff
@@ -0,0 +1,36 @@
+# Introduction
+
+### Numbers
+
+Go contains basic numeric types that can represent sets of either integer or
+floating-point values. There a different types depending on the size of value
+you require and the architecture of the computer where the application is
+running (e.g. 32-bit and 64-bit).
+
+For the sake of this exercise you will only be dealing with:
+
+- `int`: e.g. `0`, `255`, `2147483647`. A signed integer that is at least 32
+  bits in size (value range of: -2147483648 through 2147483647). But this will
+  depend on the systems architecture. Most modern computers are 64 bit therfore
+  `int` will be 64 bits in size (value rate of: -9223372036854775808 through
+  9223372036854775807).
+- `float64`: e.g. `0.0`, `3.14`. Contains the set of all 64-bit floating-point
+  numbers.
+
+For a full list of the available numeric types and more detail see the
+following resources:
+
+- [Go builtin type declarations][go-builtins]
+- [Digital Ocean - Understanding Data Types in Go][do-understanding-types]
+- [Arden Labs - Understanding Type in Go][arden-understanding-types]
+
+### Type Conversion
+
+This exercise may require using type conversion. Here is an quick overview of
```

Comment:
```suggestion
This exercise requires using type conversion. Here is an quick overview of
```

Reply:
Or, do you see a way around it?

Reply:
off the top of my head I don't, but didn't want to give too much away :smile: